platform/qt/src/qmapboxgl.cpp

```diff
@@ -806,27 +930,47 @@ void QMapboxGL::setPaintProperty(const QString& layer_, const QString& property,
     }
 }
 
+/*!
+    Returns true when the map is completely rendered, false otherwise. A partially
+    rendered map ranges from nothing rendered at all to only labels missing.
+*/
 bool QMapboxGL::isFullyLoaded() const
 {
     return d_ptr->mapObj->isFullyLoaded();
 }
 
+/*!
+    Pan the map by \a offset in pixels.
```

Comment:
Does `QPointF` or a related type specify an origin, or is it up to us to say whether a positive `y` is interpreted as an offset up or down? I see that we had to specify an origin in the `pixelForCoordinate()` documentation.